aria-practices.html

```diff
@@ -3697,6 +3697,1247 @@ <h5>Examples</h5>
+&lt;span id="download-details">PDF, 2.4 MB&lt;/span></code></pre>
+          <p>In the above example, the accessible name of the alert will be "Download PDF, 2.4 MB", with a space between "Download" and "PDF", and not "DownloadPDF, 2.4 MB".</p>
+          <ol class="warning">
+            <li>The <code>aria-labelledby</code> property cannot be chained, i.e., if an element with <code>aria-labelledby</code> references another element that also has <code>aria-labelledby</code>, the <code>aria-labelledby</code> attribute on the referenced element will be ignored.</li>
+            <li>If an element is referenced by <code>aria-labelledby</code> more than one time during a name calculation, the second and any subsequent references will be ignored.</li>
+            <li>There are certain types of elements, such as paragraphs and list items,  that should not be named with <code>aria-labelledby</code>. They are identified in the table in the <a href="#naming_role_guidance">Accessible Name Guidance by Role</a> section.</li>
+            <li>
+          If <code>aria-labelledby</code> is applied to an element with one of the roles that supports <a href="#naming_with_child_content">naming from child content</a>, content contained in the element and its descendants is hidden from assistive technology users unless it is also referenced by <code>aria-labelledby</code>.
+          It is strongly recommended to avoid using this attribute to override content of one of these elements except in rare circumstances where hiding content from assistive technology users is beneficial.
+        </li>
+            <li>Because calculating the name of an element with <code>aria-labelledby</code> can be complex and reference hidden content, testing with assistive technologies to ensure the expected name is presented to users is particularly important.</li>
+          </ol>
+        </section>
+
+        <section id="naming_with_labels">
+          <h5>Naming Form Controls with the Label Element</h5>
+          <p>
+            The HTML <code>label</code> element enables authors to identify content that serves as a label and associate it with a form control.
+            When a <code>label</code> element is associated with a form control, Browsers calculate an accessible name for the form control from the <code>label</code> content.
+          </p>
+          <p>
+            For example, text displayed adjacent to a checkbox may be visually associated with the checkbox, so it is understood as the checkbox label by users who can perceive that visual association.
+            However, unless the text is programmatically associated with the checkbox, assistive technology users will experience a checkbox without a label.
+            Wrapping the checkbox and the labeling text in a <code>label</code> element as follows gives the checkbox an accessible name.
```

Comment:
We should use labeled/labeling (although as a Brit it pains me to write it). W3C editorial guidelines on spelling are detailed at https://w3c.github.io/manual-of-style/#Spelling

Reply:
And as a Canadian, it pains me to read it. ;)
Consistency should win, though.

Although I do need to point out that the [W3C Manual of Style](https://w3c.github.io/manual-of-style/#Spelling) says:
> W3C uses Merriam-Webster's Collegiate® Dictionary, 10th Edition [[M-W](https://www.merriam-webster.com/)], on the Web as the spelling arbiter 

... and said Merriam-Webster's Collegiate® Dictionary, 10th Edition lists the following in its [definition of label](https://www.merriam-webster.com/dictionary/label):
> label **verb**
> labeled or labelled; labeling or labelling

So I have to say that it's _really_ close, but I will concede that the single-L version is listed first, and the example they give uses "labeled", so, yes, that is the one we should go with. It should probably be a separate PR, though, because it probably a document-wide change.
We just need to be careful that `aria-labelledby` always has 2 L's.

Reply:
I am correcting all instances inside the content added by this pull request so they consistently use a single 'l'. Correcting outside the naming and describing section should be a separate pull request.